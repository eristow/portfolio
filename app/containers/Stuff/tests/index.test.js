/**
 *
 * Tests for Stuff
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { render } from 'react-testing-library';
// import 'jest-dom/extend-expect'; // add some helpful assertions

import { Stuff } from '../index';

describe('<Stuff />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const dispatch = jest.fn();
    render(<Stuff dispatch={dispatch} />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Stuff />);
    expect(firstChild).toMatchSnapshot();
  });
});
