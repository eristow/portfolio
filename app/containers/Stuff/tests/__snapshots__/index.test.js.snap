// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stuff /> Should render and match the snapshot 1`] = `
<div>
  <h1>
    Other Stuff
  </h1>
  <button
    class="Button-cbu1yy-0 ksdeT"
    color="#03dac6"
    type="button"
  >
    Stock App
  </button>
</div>
`;
